### PR TITLE
(fix): fix tests after switch to consta.

### PR DESCRIPTION
**Pull request by Alexandr1111**
## Problem statement
https://artcpt.atlassian.net/browse/VEGA-4995
После обновления библиотек и замены некоторых компонентов стали падать юнит-тесты.

## Solution details
- Поправлены падающие тесты
- Изменен файл setupTests

## Type
- [ ] Feature
- [x] Bug Fix
- [ ] Refactoring
- [ ] Test
- [ ] Other

## Associated issues
https://artcpt.atlassian.net/browse/VEGA-4834

## Quality control
- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [ ] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] Docs: All the important changes and features are described
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [x] Tests: Existing unit tests passed successfully
- [ ] Tests: Existing E2E tests passed successfully
- [x] Tests: Manually tested on personal instance

## Notable statements
- [x] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

File: .huskyrc.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   hooks: {
     ...require('@gpn-prototypes/frontend-configs/.huskyrc.js').hooks,
-    'pre-push': 'yarn test',
+    // 'pre-push': 'yarn test',
```
Comment (ndrzhn) [inline .huskyrc.js:4]: надо вернуть?

**Comment by ndrzhn** [comment]
Странное название ветки, почему не по номеру задачи?

File: src/services/identity/tokenHandlers.ts [hunk 2]
```diff
@@ -25,6 +25,8 @@ function decodeToken(token: string): DecodedToken | null {
     const parts = token.split('.');
     const header = JSON.parse(atob(parts[0]));
     const payload = JSON.parse(atob(parts[1]));
+    // const header = JSON.parse(JSON.stringify(Buffer.from(parts[0], 'base64').toString('base64')));
+    // const payload =  JSON.parse(JSON.stringify(Buffer.from(parts[1], 'base64').toString('base64')));
```
[the remark this answers is not in the document]
Comment (Alexandr1111) [reply src/services/identity/tokenHandlers.ts:29]: Удалил

[the remark this answers is not in the document]
Comment (Alexandr1111) [reply src/services/identity/tokenHandlers.ts:29]: Серега на то-же ком написал))
Сейчас пушну

**Review by SergeyKatugin — APPROVED** [review]

**Pull request closed without merging**